cegs_portal/search/templates/search/v1/experiments.html

```diff
@@ -58,19 +67,33 @@
             <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" class="svg-icon mr-2"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path d="M175 389.4c-9.8 16-15 34.3-15 53.1c-10 3.5-20.8 5.5-32 5.5c-53 0-96-43-96-96V64C14.3 64 0 49.7 0 32S14.3 0 32 0H96h64 64c17.7 0 32 14.3 32 32s-14.3 32-32 32V309.9l-49 79.6zM96 64v96h64V64H96zM352 0H480h32c17.7 0 32 14.3 32 32s-14.3 32-32 32V214.9L629.7 406.2c6.7 10.9 10.3 23.5 10.3 36.4c0 38.3-31.1 69.4-69.4 69.4H261.4c-38.3 0-69.4-31.1-69.4-69.4c0-12.8 3.6-25.4 10.3-36.4L320 214.9V64c-17.7 0-32-14.3-32-32s14.3-32 32-32h32zm32 64V224c0 5.9-1.6 11.7-4.7 16.8L330.5 320h171l-48.8-79.2c-3.1-5-4.7-10.8-4.7-16.8V64H384z"/>
             </svg> Multi-Experiment Coverage Information
         </div>
-        <div class="mt-5">The coverage data from the following experiments have been intersected based on their
-            <select name="views" id="feature-select">
-                <option value="sources">Tested Elements</option>
-                <option value="targets">Genes</option>
-                <option value="sources-targets" selected>Tested Elements and Genes</option>
-            </select></div>
-        <ul class="list-disc mt-5">
-            {% for expr in experiments %}
-            <li>{{ expr.name }} <a class="link" href="{% url 'search:experiment' expr.accession_id %}"><svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="currentColor" class="svg-link-arrow bi bi-arrow-up-right-square-fill inline" viewBox="0 0 16 16">
-                <path fill-rule="evenodd" d="M14 0a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2zM5.904 10.803 10 6.707v2.768a.5.5 0 0 0 1 0V5.5a.5.5 0 0 0-.5-.5H6.525a.5.5 0 1 0 0 1h2.768l-4.096 4.096a.5.5 0 0 0 .707.707z"/>
-            </svg></a></li>
-            {% endfor %}
-        </ul>
+        <table class="min-w-full">
```

Comment:
I don' think this should be a table. Or, if you'd like it to be, only the list of experiments should be in the table, not the intersection selector line. Also, was the asterisk added on purpose?

Reply:
Removed from table. Asterisk was added

Reply:
Why did you add the asterisk?

Reply:
Deleted